### DISCUSSION TITLE
Items in rectangle you are dragging

### DISCUSSION THREAD

**Discussion by edmana** [react-grid-layout/react-grid-layout#1472] [Q&A]
We are working w/ this and finding that when we have fields defined within the rectangle we are dragging, the field no longer behaves like on a regular page. For example, if I try to click into the field and drag highlight the text, the draggable component takes over instead.

Any idea how to correct this? One option we thought of was to use an icon in the top left and make that the "draggable" part of the rectangle in order to move it around but that doesn't seem native to the component.

Thanks!
Ed

**Comment by STRML** [comment]
See the handle prop.

**Reply by edmana** [reply]
I took a look and I wonder if I explained it wrong. The handle prop looks like displaying/locating the icon to extend the widgets in various directions. My issue is when I click inside the widget to drag it and move it around, if I have fields in the widget, the click on the field allows me to drag vs. going into the field to say highlight it or use the field.

Also, in terms of the icon (arrow) for re-sizing, can that icon be changed to say a material-ui icon?

Thanks!
